core/spec.md

```diff
@@ -62,12 +68,12 @@ Resources represent the main data of interest for the Registry. In the
 filesystem analogy, these would be the "files". A Resource exist under a
 single Group and, similar to Groups, have a set of Registry metadata.
 However, unlike a Group which only has Registry metadata, Resources can
-have an associated "document" associated with it. This specification places
-no restriction on the type of content stored in the Resource's document.
+have a "document" associated with it. For example, a "schema` Resource might
```

Comment:
The "it" here is a little odd, as Resources is plural. (Pre-existing issue.)

Reply:
(Perhaps "each Resource can have a "document" associated with it"?

Reply:
done